ddl/index.go

```diff
@@ -657,9 +656,11 @@ func makeupIndexColFieldMap(t table.Table, indexInfo *model.IndexInfo) map[int64
 
 // splitTableRanges uses PD region's key ranges to split the backfilling table key range space,
 // to speed up adding index in table with disperse handle.
-func (d *ddl) splitTableRanges(t table.Table, startHandle int64) ([]kv.KeyRange, error) {
+func (d *ddl) splitTableRanges(t table.Table, reorgInfo *reorgInfo) ([]kv.KeyRange, error) {
+	startHandle := reorgInfo.Handle
+	endHandle := reorgInfo.ReorgMeta.EndHandle
```

Comment:
Can we put EndHandle in ReorgInfo?

Reply:
Please address this comment.

Reply:
reorgInfo embedded *model.Job, so there is no need to store endHandle in reorgInfo.